code/datums/mutations/fire_breath.dm

```diff
@@ -0,0 +1,94 @@
+/datum/mutation/human/firebreath
+	name = "Fire Breath"
+	desc = "An ancient mutation that gives lizards breath of fire."
+	quality = POSITIVE
+	difficulty = 12
+	locked = TRUE
+	text_gain_indication = "<span class='notice'>Your throat is burning!</span>"
+	text_lose_indication = "<span class='notice'>Your throat is cooling down.</span>"
+	power_path = /datum/action/cooldown/spell/cone/staggered/fire_breath
+	instability = 30
+	energy_coeff = 1
+	power_coeff = 1
+
+/datum/mutation/human/firebreath/modify()
+	. = ..()
+	var/datum/action/cooldown/spell/cone/staggered/fire_breath/to_modify = .
+	if(!istype(to_modify)) // null or invalid
+		return
+
+	if(GET_MUTATION_POWER(src) <= 1) // we only care about power from here on
+		return
+
+	to_modify.cone_levels += 2  // Cone fwooshes further, and...
+	to_modify.self_throw_range += 1 // the breath throws the user back more
+
+	return .
```

Comment:
i get you're doing this for readabiliy sake, but its consistent with the other returns in this proc